pkg/storage/unified/resource/resource.proto

```diff
@@ -399,6 +399,9 @@ message ResourceSearchRequest {
 
   // explain each result (added to the each row)
   bool explain = 9;
+
+  // filters
+  repeated string filters = 10;
```

Comment:
the ListOptions already has typed filters for labels and fields...  This lets us avoid needing to parse the filter

Reply:
👍  updated